sentry_sdk/integrations/aiohttp.py

```diff
@@ -258,12 +264,23 @@ async def on_request_start(session, trace_config_ctx, params):
                     params.headers[key] = value
 
         trace_config_ctx.span = span
+        trace_config_ctx.span_data = data
```

Comment:
Nit: maybe let's call it something like `request_data` instead of `span_data` since it's also used for breadcrumbs? (Also in the other integrations)